crates/bevy_winit/src/winit_windows.rs

```diff
@@ -95,8 +102,14 @@ impl WinitWindows {
                 winit_window_builder.with_min_inner_size(min_inner_size)
             };
 
+        let window_title = window_descriptor.title.as_str();
+
         #[allow(unused_mut)]
-        let mut winit_window_builder = winit_window_builder.with_title(&window_descriptor.title);
+        // Due to a UIA limitation, winit windows need to be invisible for the
```

Comment:
Can you clarify what this comment means? 

Would this issue go away if accesskit integration wasn't relying on winit?

Reply:
For a window that implements UI Automation, the window procedure must be able to handle a `WM_GETOBJECT` message by returning a UIA provider at the moment that the window is shown. This is because UIA Core running in the client (e.g. screen reader) process is notified at the moment that a new window is shown, tries to get a native UIA provider for that window, and uses the result of that request to determine how it should register UIA event handlers. If the window is shown, and then UIA support is injected later, handling of events, such as live regions or changes in the state of dynamic UI elements like check boxes, will be erratic, depending on the screen reader. We've seen it happen; we know it's a real problem.

If winit had a way of registering a `WM_GETOBJECT` handler when the window is created, then this wouldn't matter. As it is now, the `accesskit_winit` crate has to use a Win32 workaround known as window subclassing. (It uses a similar workaround on macOS, but I don't know if doing so after the window is already visible is a problem on that platform.) So the window is created by winit, then the AccessKit winit adapter injects UIA support via subclassing. Because we have to use that workaround, it's necessary to create the window invisible, then do the injection, then show the window.